backend/src/govsg/middlewares/govsg-verification.middleware.ts

```diff
@@ -4,16 +4,39 @@ import { GovsgMessage } from '@govsg/models'
 import { GovsgVerification } from '@govsg/models/govsg-verification'
 import { WhatsAppApiClient } from '@shared/clients/whatsapp-client.class/types'
 import { sendPasscodeCreationMessage } from '@govsg/services/govsg-verification-service'
+import { Op } from 'sequelize'
+
+const generateSearchOptions = (search: string) => {
+  // TODO: Use an OR operation
+  if (!search) {
+    return {}
+  } else if (search.match(/^\d{1,10}$/)) {
+    return {
+      recipient: {
+        [Op.like]: `%${search}%`,
+      },
+    }
+  }
+  return {
```

Comment:
nit: Should we complete the if-elseif-else structure?

Reply:
Thanks for catching the `else if`. I personally prefer `ifs` and early exit. Let me change the `else if` instead.